src/app/core/upload/upload.service.ts

```diff
@@ -73,7 +72,6 @@ export class Upload {
     this.uuid = UUID.UUID();
     this.name = file.name;
     this.size = file.size;
-    this.duration = file['duration'];
```

Comment:
Instead of setting the duration on the `Upload` and `File` ... it's now set on the `AudioFileModel` along with bitrate/frequency/etc.  Which makes way more sense.